waterdrop.gemspec

```diff
@@ -16,7 +16,7 @@ Gem::Specification.new do |spec|
   spec.description   = spec.summary
   spec.license       = 'MIT'
 
-  spec.add_dependency 'delivery_boy', '~> 0.2'
+  spec.add_dependency 'delivery_boy', '>= 0.2', '<= 2.x'
```

Comment:
The folks over at zendesk opted to do a 1.0 release of delivery boy instead of a patch version bump. None of the changes appear to be breaking, so I opted to adjust the dependency to include 1.x releases: https://github.com/zendesk/delivery_boy/blob/a8c7b0de05a7d97054365da910f9f651046ea206/CHANGELOG#L3-L7

Reply:
They mentioned it (the 1.0 release) was overdue: https://github.com/zendesk/delivery_boy/pull/50#issuecomment-566983507

Reply:
(force pushed to change `<= 2.x` to `< 2.x`)